sdcm/cluster.py

```diff
@@ -1817,11 +1818,14 @@ def download_scylla_manager_repo(self, scylla_repo):
         if not self.is_rhel_like():
             self.remoter.run(cmd="sudo apt-get update", ignore_status=True)
 
-    def install_manager_agent(self, auth_token, manager_scylla_repo, package_path=None):
+    def install_manager_agent(self, package_path=None):
+        auth_token = Setup.test_id()
```

Comment:
Where the part of the manager server end, that use the test_id too ?

Reply:
updated... now the function that the manager server uses is getting Setup.test_id too.